heartbeat/monitors/logger/logger.go

```diff
@@ -96,6 +97,11 @@ func extractRunInfo(event *beat.Event) (*MonitorRunInfo, error) {
 		errors = append(errors, err)
 	}
 
+	attempts, err := event.GetValue("summary.attempt")
+	if err != nil {
+		errors = append(errors, err)
```

Comment:
the problem with appending errors that noticed was, it will be generic like `key not found` which doesnt give any context. 

We should wrap probably this error with more meaningful message instead.